src/css/archwizard.scss

```diff
@@ -1,11 +1,127 @@
-// color definitions
-$wz-color-default: #E6E6E6 !default;
-$wz-color-current: #808080 !default;
-$wz-color-done: #339933 !default;
-$wz-color-optional: #38ef38 !default;
-$wz-color-editing: #FF0000 !default;
+// Wizard layouts to be generated (all by default).
+// Override this variable to exclude layouts you don't need.
+$wz-layouts: 'horizontal' 'vertical' !default;
+
+// Wizard indicator styles to be generated (all by default).
+// Override this variable to exclude indicator styles you don't use.
+$wz-indicator-styles: 'small', 'large-filled', 'large-empty', 'large-filled-symbols', 'large-empty-symbols' !default;
+
+// Color definitions
+//
+// This is a mapping from indicator style and wizard step state to colors.

```

Comment:
`This` as in `the color definitions`?

Reply:
In 6202716 I squashed the two lines into

```
// Color definitions - a mapping from indicator style and wizard step state to colors.
```

This way it is more concise.